test/functional/bookmarklet.js

```diff
@@ -25,6 +25,34 @@ describe('BookmarkletController', () => {
 
       let responseData = await response.json()
       responseData.should.have.property('posts')
+      responseData.should.have.property('subscriptions')
+      responseData.subscriptions.should.have.length(1)
     })
+
+    it('should allow create posts in multiple feeds', async () => {
+      let group = await createGroupAsync(luna, 'new-shiny-group')
+
+      let response = await createPostViaBookmarklet(luna, 'Hello, world!', null, null, [luna.username, group.username])
+      response.status.should.eql(200)
+
+      let responseData = await response.json()
+
+      responseData.should.have.property('posts')
+      responseData.should.have.property('subscriptions')
+      responseData.subscriptions.should.have.length(2)
+    })
+
+    it('should ignore nonexistent groups', async () => {
+      let response = await createPostViaBookmarklet(luna, 'Hello, world!', null, null, [luna.username, 'non-existent-group'])
+      response.status.should.eql(200)
+
+      let responseData = await response.json()
+
+      responseData.should.have.property('posts')
+      responseData.should.have.property('subscriptions')
+      responseData.subscriptions.should.have.length(1)
+    })
```

Comment:
I'm not 100% sure that ignoring is the best approach here. Explicit arrow would indicate that something went wrong. 

Ignoring might lead to unexpected result in case client-software is wrong. User thinks she posted to 3 feeds, but only one of those really got the message.